store/src/state.rs

```diff
@@ -411,6 +432,7 @@ impl State {
         Ok((latest, peaks, account_states))
     }
 
+    /// Returns data needed by the block producer to verify transactions proofs.
```

Comment:
I would say: "Returns the data needed by the block producer to verify transaction validity." Because this data is not really needed to verify the proofs but rather that the proofs apply to valid account/nullifier states.

Reply:
changed